src/test/java/com/google/javascript/gents/singleTests/inner_types.ts

```diff
@@ -0,0 +1,28 @@
+
+export class MyClass {
+  /** @export type property*/
+  type: Type1;
+
+  /**
+   * Constructor for MyClass
+   */
+  constructor(data: {type: Type1}) {
+    this.type = data.type;
+  }
+
+  /**
+   * equal function
+   * @export
+   */
+  equals(otherData: MyClass): boolean {
+    return this.type.a === otherData.type.a;
+  }
+}
+
+export type Type1 = {
```

Comment:
Looked through some code in google3 and found that the references in other files are quite common.
It's currently not handled in this PR. With this PR only references in the same file are renamed.

I'm planning to work on a separate PR for that, along with some multitests.